tests/test_models.py

```diff
@@ -194,3 +205,38 @@ def test_energy_dipole_mace():
         np.array(rot @ output["dipole"][0].detach().numpy()),
         output["dipole"][1].detach().numpy(),
     )
+
+
+def test_scale_shift_dipole_mace(dipole_model_config, data_batch_1):
+    dipole_model_config.update({"atomic_inter_scale": 2.0, "atomic_inter_shift": 1.0})
+
+    # create dipole MACE model
+    model = modules.ScaleShiftEnergyDipoleMACE(**dipole_model_config)
+
+    output = model(
+        data_batch_1,
+        training=True,
+    )
+
+    for key in ["energy", "node_energy", "forces", "dipole", "atomic_dipoles"]:
+        assert key in output
+
+
+def test_scaled_and_shifted(dipole_model_config, data_batch_1):
```

Comment:
While the tests were enormously useful while refactoring, there is more to test on all these classes. Should be somewhat easier now.